cylc/flow/scripts/tui.py

```diff
@@ -18,8 +18,13 @@
 
 View and control running suites in the terminal.
 
-(TUI = Terminal User Interface)
+(Tui = Terminal User Interface)
+
+WARNING: Tui is experimental and may break with large flows.
```

Comment:
Maybe flag that we expect better performance in the near-ish future? Something like this:
```suggestion
WARNING: Tui is experimental and may break with large flows. An upcoming change to the
way Tui receives data from the scheduler will make it much more efficient in the future.
```